feature-libs/user/profile/core/facade/facade-providers.ts

```diff
@@ -29,6 +32,6 @@ export const facadeProviders: Provider[] = [
   },
   {
     provide: UserRegisterFacade,
-    useExisting: UserRegisterService,
+    useExisting: environment.cdc? CDCUserRegisterService: UserRegisterService,
```

Comment:
As above. I think this should be handled somehow inside CDC lib. To not touch any other file outside CDC integration lib.